src/native/src/3rdparty/json/ChangeLog.md

```diff
@@ -0,0 +1,224 @@
+# Change Log
```

Comment:
I was talking with dethraid about this. Personally I don't want to have the whole history of another repo included since it's not really needed, and it just increases the time that it takes onto clone the repo to be ready to build. That's the main reason really. But I haven't been including all dependencies this way. Just the single file style headers that you just throw in your project. Though I happened to fork one of them integrated some changes that never made it to the official repo.